apps/src/code-studio/pd/foorm/editor/components/FoormEntityEditor.jsx

```diff
@@ -39,27 +34,30 @@ const styles = {
 const PREVIEW_ON = 'preview-on';
 const PREVIEW_OFF = 'preview-off';
 
-class FoormEditor extends React.Component {
+// Interface for admins to try out Foorm "entity" (either a form or library question) in real-time.
```

Comment:
NIT (don't worry about changing): i used the wrong terminology when i asked for a "multiline" comment. i meant a block comment, e.g.:
```
/**
* my text
* /
```
generally that's the preferred format for class comments and function comments. `//` is supposed to just be for inline comments.